src/player_activity.cpp

```diff
@@ -137,13 +138,13 @@ void player_activity::do_turn( player *p )
             break;
         case ACT_PICKAXE:
             // Based on speed, not time
-            moves_left -= p->moves;
+            moves_left -= (int)(p->moves * season_mult);
```

Comment:
`(int)` is a C-style cast and will cast basically everything to everything. `static_cast` is more restricted, it will only cast things that have a known conversions (e.g. float to int, or int to float ...). Further reading: http://stackoverflow.com/questions/332030/when-should-static-cast-dynamic-cast-const-cast-and-reinterpret-cast-be-used